pandas/core/strings.py

```diff
@@ -2466,7 +2472,31 @@ def cat(self, others=None, sep=None, na_rep=None, join="left"):
         2    -c
         dtype: object
 
-        For more examples, see :ref:`here <text.concatenate>`.
+        If `others` is a Series/Index/DataFrame, index alignment is
```

Comment:
This and the next paragraph duplicate what was said above. If there's a way to consolidate and make more concise would be preferable